ups/obs_auxTel.table

```diff
@@ -0,0 +1,14 @@
+setupRequired(afw)
```

Comment:
I think only modules that are actually imported need to be in this table file.  For example, `skypix` is not used anywhere in this module.  I suspect there are others.